### ISSUE TITLE
Preparing the ALSA backend for PipeWire

### ISSUE THREAD

**Issue by mxa** [pure-data/pure-data#1487]
PipeWire can simulate jack and pulseaudio servers or ALSA interfaces for applications to connect to. Pure Data can use PipeWire through the jack or pulseaudio backends, but not through ALSA, because Pure Data is refusing to talk to software devices. This may have been a sensible decision back then, but in the advent of PipeWire it should be revisited.

**Comment by umlaeute** [comment]
i think this should be thought of as unrelated to pipewire.

in any case, i think a sensible compromise would be a configuration option to enable/disable software (that is: non-hardware) devices.

**Comment by mxa** [comment]
I agree this can be thought of as unrelated to Pipewire. The issue arose from the discussion which backend a PipeWire user should preferably select.
